Use the correct filename for cert bundle from ca-certificates package.

---
 tls/Makefile.am |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

Index: libressl-4.4.0/tls/Makefile.am
===================================================================
--- libressl-4.4.0.orig/tls/Makefile.am
+++ libressl-4.4.0/tls/Makefile.am
@@ -54,13 +54,13 @@ libtls_la_LIBADD += $(PLATFORM_LDADD)
 
 libtls_la_CPPFLAGS = $(AM_CPPFLAGS)
 if OPENSSLDIR_DEFINED
-libtls_la_CPPFLAGS += -DTLS_DEFAULT_CA_FILE=\"@OPENSSLDIR@/cert.pem\"
+libtls_la_CPPFLAGS += -DTLS_DEFAULT_CA_FILE=\"@OPENSSLDIR@/ca-bundle.pem\"
 else
 if HOST_WIN
 # See crypto/Makefile.am: keep the default CA file out of a drive-relative path.
-libtls_la_CPPFLAGS += -DTLS_DEFAULT_CA_FILE=\"C:/Windows/libressl/ssl/cert.pem\"
+libtls_la_CPPFLAGS += -DTLS_DEFAULT_CA_FILE=\"C:/Windows/libressl/ssl/ca-bundle.pem\"
 else
-libtls_la_CPPFLAGS += -DTLS_DEFAULT_CA_FILE=\"$(sysconfdir)/ssl/cert.pem\"
+libtls_la_CPPFLAGS += -DTLS_DEFAULT_CA_FILE=\"$(sysconfdir)/ssl/ca-bundle.pem\"
 endif
 endif
 
